index.js

```diff
@@ -45,5 +49,26 @@ module.exports = function (input, opts) {
 		}
 	});
 
+	if ('_' in input &&
+		!(Array.isArray(opts.excludes) && opts.excludes.indexOf('_') !== -1) &&
+		!(Array.isArray(opts.includes) && opts.includes.indexOf('_') === -1)) {
```

Comment:
Can you simplify this to not be double negative (You have both `!(` and `!==`)?

This check needs to be tested.